docs/styles/styledocUtils.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FDS from 'lib/dictionary/js/styleConstants';
+
+export const generalStyles = {
```

Comment:
ehhh so this is really difficult, and was probably one of the more difficult parts of creating these examples.... essentially, which styles are shared, which styles serve as good defaults, etc. If you notice, initially I made some of these layout components in`styledocUtils` that I use "sometimes"... but quickly notice there were instances where I needed to use the raw styles.

It's tough because I sort of don't want to make component abstractions until I'm "sure" that they're a worthy abstraction. I borderline wanted to go and rip out the components as even the styles aren't that consistent and I end up overriding them to the point where I question their value and general applicability.

But yeah it's rough because I could go in here and create an abstraction that makes more sense, but it's like... 

1. How much time does it makes sense to dedicate to that, given the nature of the work.
2. How many of these abstractions will even stick around? So I'm increasingly pondering the utility of putting the examples INSIDE the tables, so that users don't have to mentally draw a line between the the class, and the example below it that repeats the label. If that eventually happens, it will nullify all of these styles.
3. Even if I create the right abstraction, I'm not totally confident, in this scenario, that it will hold up to scrutiny if we add further examples.

Given all that, I sort of just left it in a mediocre, pretty good, not great but not bad state given the ephemeral state of the current docs. Happy to change it though if you're interested in it.

Reply:
Sounds like not worth the effort for this round